### PR TITLE
Feature/add shift review page

### PR DESCRIPTION
**Pull request by Meleeman01**
this is the first iteration and it functions as described, with one minor difference being that the shift name is not in the table instance, but rather a heading above the table showing users who have volunteered for slots. department leads and admins can access this page and change the rating of each volunteer who volunteered for a shift.

### CODE REVIEW

**Review by itsrachelfish — DISMISSED** [review]
Shifts without volunteers should be skipped.

Changes should be auto-saved instead of prompting the user to save each dropdown separately. 

Screenshot from 2019-06-24 03-19-32

**Comment by Meleeman01** [comment]
This is what it looks like now after the Edits requested by Rachel, I decided to add a small notification that simply says "saved!" and takes advantage of the existing code in admin.js and elements on the page.
saveToastNotification

**Comment by demogorgonzola** [comment]
Everything looks great!

Think about putting in some logic to replace the "Saved!" message with a response dependent on the fetches response. That way no one's being mislead by the response incase something goes wrong.

File: laravel/resources/js/admin.js [hunk 1]
```diff
@@ -118,11 +118,46 @@ $(document).ready(function()
...
+            _token: csrf
+        };
+
+        ajaxOptions.body = JSON.stringify(data);
+        fetch('/slot/'+ slot +'/edit', ajaxOptions);
+
+        var message = $(this).parents('.volunteer').find('.toast-message').style({visibility: 'visible', opacity: 1});
+        setTimeout(function()
+        {
+            clearTimeout()
```
Comment (itsrachelfish) [inline laravel/resources/js/admin.js]: This doesn't actually do anything. `clearTimeout` requires an argument (a reference to the timeout being cleared)

Comment (Meleeman01) [reply laravel/resources/js/admin.js]: alright i fixed it to where it actually does that, how silly of me to miss that

**Review by kristenward — APPROVED** [review]
This seems to be working as expected now. 

Changes since last review:
- Merge in updated master branch
- Update lock file
- reconcile change from old version of master (see comment)

@itsrachelfish please review once again

**Review by itsrachelfish — APPROVED** [review]
Not the prettiest looking page, but it gets the job done. Tested with a database dump from the Apogaea production server, works as expected! :+1: 

Screenshot from 2021-08-23 17-59-51

**Pull request merged**